local-dev/templates/firecloud-orchestration.conf.ctmpl

```diff
@@ -143,10 +143,3 @@ staticNotebooks {
   baseUrl = "https://calhoun.dsde-dev.broadinstitute.org"
 }
 
-importService {
-  server = "https://terra-importservice-dev.appspot.com"
-  bucketName = "import-service-batchupsert-dev"
-  enabled = true
-}
```

Comment:
do you have a corresponding terra-helmfile PR to remove this stanza?

Reply:
[github.com/broadinstitute/terra-helmfile/pull/5538](https://github.com/broadinstitute/terra-helmfile/pull/5538/files)